src/AutoRest.CSharp/Properties/AssemblyInfo.cs

```diff
@@ -3,4 +3,5 @@
 
 using System.Runtime.CompilerServices;
 
-[assembly:InternalsVisibleTo("AutoRest.TestServer.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
\ No newline at end of file
+[assembly:InternalsVisibleTo("AutoRest.TestServer.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
+[assembly:InternalsVisibleTo("AutoRest.TestServerLowLevel.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
```

Comment:
Curious, what internals do we use?

Reply:
test\AutoRest.TestServer.Tests\Infrastructure\JsonAsserts.cs(11,65): error CS0122: 'IUtf8JsonSerializable' is inaccessible due to its protection level 
test\AutoRest.TestServer.Tests\Infrastructure\JsonAsserts.cs(25,52): error CS0122: 'IUtf8JsonSerializable' is inaccessible due to its protection level
test\AutoRest.TestServer.Tests\Infrastructure\TestServerTestBase.cs(23,25): error CS0122: 'ClientDiagnostics' is inaccessible due to its protection level